src/solc_0.7/common/BaseWithStorage/ERC721BaseToken.sol

```diff
@@ -248,17 +223,16 @@ contract ERC721BaseToken is IERC721, WithSuperOperators, WithMetaTransaction {
         }
     }
 
-    /**
-     * @notice Check if the contract supports an interface
-     * 0x01ffc9a7 is ERC-165
-     * 0x80ac58cd is ERC-721
-     * @param id The id of the interface
-     * @return True if the interface is supported
-     */
+    /// @notice Check if the contract supports an interface.
+    /// 0x01ffc9a7 is ERC-165.
+    /// 0x80ac58cd is ERC-721
+    /// @param id The id of the interface.
+    /// @return whether the interface is supported.
     function supportsInterface(bytes4 id) public pure virtual override returns (bool) {
         return id == 0x01ffc9a7 || id == 0x80ac58cd;
     }
 
+    /// @dev See transferFrom.
```

Comment:
👍 
Is this something that we are applying to all internal functions where the name is effectively replicated?

Reply:
I think this convention makes sense if the internal function just repeats the name and same params, otherwise additional comments are probably warranted?

Reply:
I like the convention, works for me!